### DISCUSSION TITLE
XDrip Sync Follower unreliable.

### DISCUSSION THREAD

**Discussion by elgrupo123** [NightscoutFoundation/xDrip#3235] [General]
XDrip Sync Follower unreliable.
We have been having recurring problems on our followers' devices for several weeks. The values are sometimes not available for up to 1 hour. If I briefly change the data source and switch back to Xdrip Sync Follower, the values are back immediately.
Does anyone have similar problems?

**Comment by Navid200** [comment]
>  If I briefly change the data source and switch back to Xdrip Sync Follower, the values are back immediately.

What version of xDrip are you using?
If a Nightly, do you experience the same if you go back to the stable release as well?

**Reply by elgrupo123** [reply]
It is always the latest Nightly. But it's a good idea, I'll go back to the stable one and keep watching.

**Reply by Navid200** [reply]
If the problem is resolved after trying the stable release, then, please try this Nightly: https://github.com/NightscoutFoundation/xDrip/releases/tag/2023.12.05

**Reply by elgrupo123** [reply]
The problem is still there with both versions (stable or from 5.12.23). Maybe this is the reason for this error message?
![PXL_20231212_165251492](https://github.com/NightscoutFoundation/xDrip/assets/92778761/2c7c72b8-3c9d-43b8-a5e9-88f1df31ae33)

**Comment by elgrupo123** [comment]
It's been working again without any problems since yesterday evening, only this error message appeared last night.
![Screenshot_20231213-150702](https://github.com/NightscoutFoundation/xDrip/assets/92778761/961d3920-3077-4229-b642-5cbf7fcc5c18)
